Examples/repeat/main.swift

```diff
@@ -12,6 +12,9 @@
 import ArgumentParser
 
 struct Repeat: ParsableCommand {
+    // Optional abstracts and discussions are used for help output.
+    static let configuration = CommandConfiguration(discussion: "Utility to repeat an phrase a number of times")
+
```

Comment:
Could you move this test into 'ArgumentParserUnitTests/HelpGenerationTests.swift'?

Reply:
I'll give it a try. I moved it out because all the HelpGenerationTests didn't show the top elements (namely OVERVIEW) so I ended up using one of the example files.

Reply:
I think you'll like the new test better.